pkg/compact/compact.go

```diff
@@ -958,16 +958,18 @@ func (c *BucketCompactor) Compact(ctx context.Context) (rerr error) {
 		}
 
 		level.Info(c.logger).Log("msg", "start of compactions")
+		st := time.Now()
+		compactCount := 0
 
 		// Send all groups found during this pass to the compaction workers.
 		var groupErrs terrors.MultiError
-	groupLoop:
 		for _, g := range groups {
 			select {
 			case groupErr := <-errChan:
 				groupErrs.Add(groupErr)
-				break groupLoop
 			case groupChan <- g:
+				compactCount++
+				level.Info(c.logger).Log("msg", "start compact group", "group", g.Key(), "compactRatio", fmt.Sprintf("%d/%d", compactCount, len(groups)), "duration", time.Since(st))
```

Comment:
I'm OK adding a log at the beginning of a group compaction, but this is not the right place to log the begin of a compaction. It should be logged at beginning of `Group.Compact()`. I'm also not much sure about the value of `compactRatio` and `duration` to be honest. What's the use case for that?

Reply:
yeah, it's not a message for group, it's  for log for compact loop, very useful for lot's of groups and system manager have no idea the process of compact
- compactRation may be confuse `compact`, i will change to `progressRate`
- remove `duration`, and add a `finish of compactions` log message

thanks for your comment again